cmd/access/node_builder/access_node_builder.go

```diff
@@ -929,6 +934,27 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 
 	if builder.executionDataSyncEnabled {
 		builder.BuildExecutionDataRequester()
+		if builder.stateStreamListenAddr != "" {
+			builder.Component("state stream engine", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
```

Comment:
nit: rename this to something like "exec state stream engine" so we're explicit about what kind of data we're exposing.